src/main/kotlin/racingcar/domain/Validator.kt

```diff
@@ -0,0 +1,20 @@
+package racingcar.domain
+
+object Validator {
+    const val NAME_LENGTH_LIMIT = 5
+
+    /** 주어진 자동차 이름 [name]을 검증합니다. */
+    fun validateCarName(name: String) : String {
```

Comment:
공백과 관련하여 컨벤션 위반이 있는 것 같아요!

```suggestion
    fun validateCarName(name: String): String {
```

Reply:
모든 파일에 코드 자동 포맷을 적용했다고 생각했는데, 이 파일은 누락된 것 같습니다. 알려주셔서 감사합니다!